cmd/deck/main.go

```diff
@@ -823,6 +827,34 @@ func handleProwJobs(ja *jobs.JobAgent, log *logrus.Entry) http.HandlerFunc {
 	}
 }
 
+func handleConfiguredJobs(o options, cfg config.Getter, log *logrus.Entry) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		jobConfig := cfg().JobConfig
+		setHeadersNoCaching(w)
+		p := r.URL.Path
+		if strings.HasSuffix(p, "configured-jobs/") { // If there is no further path, fetch the index page
+			index := GetIndex(jobConfig)
+			handleSimpleTemplate(o, cfg, "configured-jobs-index.html", index)(w, r)
+		} else {
```

Comment:
Instead of else I would return. Also I think this function needs to be a little bit more cautious around splitting and print errors if what is expected is not really there.